scaleway/helpers_k8s.go

```diff
@@ -88,6 +89,50 @@ func waitK8SClusterDeleted(k8sAPI *k8s.API, region scw.Region, clusterID string)
 	return fmt.Errorf("Cluster %s has state %s, wants %s", clusterID, cluster.Status.String(), k8s.ClusterStatusDeleted.String())
 }
 
+func waitK8SPoolReady(k8sAPI *k8s.API, region scw.Region, poolID string) error {
+	_, err := k8sAPI.WaitForPool(&k8s.WaitForPoolRequest{
+		PoolID:  poolID,
+		Region:  region,
+		Timeout: scw.DurationPtr(K8SPoolWaitForReadyTimeout),
+	})
+
```

Comment:
Check if `pool.Status == PoolStatusReady`, return an error if it is a `PoolStatusWarning`

Reply:
Done the same way as ```waitK8SClusterReady```